lib/filters/validator.js

```diff
@@ -43,44 +42,44 @@ const supportedValidators = ['maximum',
  * @constructor
  */
 class Validator {
-    constructor(parameters, definitions) {
+    constructor(parameters = [], requestBody = {}, schemas) {
```

Comment:
The name 'requestBody' is a little confusing. It's the req body spec, not the body itself right? Can we make it less confusing?

Reply:
So, here we assume that only 'schemas' out of components will be used in parameter definitions (like we do now in RESTBase) for example. Starting to use `parameters` components seem nice, we have a lot of shared parameters like `title` or `accept` between endpoitns, so abstracting that out would be neat. However, these 2 PRs are already enormous, so we can followup to do that in the next iteration. For now, would you mind adding a comment that that is not supported?

Reply:
IDK `requestBody` is what is used in the swagger docs and spec so it seems appropriate to follow that name. 

I did make a note that parameters and requestBodies are not currently supported.